aws.yaml

```
@@ -1,5 +1,65 @@
 apiVersion: release.giantswarm.io/v1alpha1
 kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  name: v12.0.1
+spec:
+  state: active
+  date: 2020-05-11T15:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: cert-manager
+    componentVersion: 0.9.0
+    version: 1.0.7
+  - name: chart-operator
+    version: 0.12.4
+  - name: cluster-autoscaler
+    componentVersion: 1.16.2
+    version: 1.1.4
```

```suggestion
    componentVersion: 1.16.5
    version: 1.16.0
```

If you'd like to test the latest cluster-autoscaler version along the way...